appengine/gophers/gophers-4/main.go

```diff
@@ -8,10 +8,11 @@ import (
 	"fmt"
 	"html/template"
 	"net/http"
-	"time"
```

Comment:
What if we move the START to above `"time"` here? That would mean the tutorial says to add `"google.golang.org/appengine"` when it's already been used in the earlier parts, but it's good to keep the group of standard library imports.

Reply:
sgtm!